### PR TITLE
Table of contents added to README

### PR DESCRIPTION
**Pull request by MartyO256**

### CODE REVIEW

**Comment by robmadole** [comment]
@MartyO256 thanks for taking the time to put this together. However, what we were going for is an integration more similar to [this](https://github.com/FortAwesome/vue-fontawesome/commit/3b1b48349b0bae1357a404157179cd54e7a06d4d) that uses `husky`, `lint-staged`, and `markdown-toc`. We want this TOC to be easy to maintain. Are you interested in working on this and updating this PR?

**Comment by MartyO256** [comment]
It linted the whole README.md. This may be conflicting with #90.

File: package.json [hunk 1]
```diff
@@ -20,12 +20,17 @@
     "dist": "cross-env NODE_ENV=production npm run build",
     "lint": "eslint src",
     "prettier": "pretty-quick src --staged",
-    "precommit": "yarn lint && yarn prettier",
+    "precommit": "yarn lint && yarn prettier ; lint-staged",
```
Comment (robmadole) [inline package.json]: Small tweak here. Let's do `yarn lint && yarn prettier && lint-staged`

File: README.md [hunk 2]
```diff
@@ -4,6 +4,20 @@
 
 Font Awesome 5 React component
 
+<!-- toc -->
```
Comment (robmadole) [inline README.md]: Perfect!

**Comment by robmadole** [comment]
Very nice!

**Pull request merged**